### ISSUE TITLE
Connected but no Allocs

### ISSUE THREAD

**Issue by user00101** [Storj/storjshare-gui#739]
- **GUI Version:** v7.2.0
- **Operating System:** Windows 7

### Expected Behavior

Please describe the program's expected behavior.

```
Storjshare should be receiving Allocs.
```

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

```

 Client is connected but no longer receives Allocs and it has been like this a few weeks even after repeated installs/restarts.
```

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. ...
2. ...
3. ...

### Screenshots (Optional)
![capture](https://user-images.githubusercontent.com/33013843/33448862-1d59ca5c-d5d5-11e7-8817-0c02b66c73a8.PNG)

If the error is graphical in nature it is helpful to provide a screenshot.

**Comment by littleskunk** [comment]
Logfiles please

**Comment by user00101** [comment]
Hopefully this helps. 

[ce979501f9e793ec3b18facaf9eb65be222b7dc0_2017-11-30.log](https://github.com/Storj/storjshare-gui/files/1519257/ce979501f9e793ec3b18facaf9eb65be222b7dc0_2017-11-30.log)

**Comment by littleskunk** [comment]
Port 4000 is closed on 107.140.18.141
